Scene room

GGameover303 opened the pull request.

GGameover303 committed:
Talk With Oldlady

GGameover303 committed:
oldladystillhere = false

GGameover303 committed:
เล่นได้ถึงออกจาก Supermarket

GGameover303 committed:
Can OpenBox After Work

GGameover303 committed:
แสดงกล่องถ้าเลือกไม่เปิดแม้เปลี่ยนซีน

- กล่องจะเปิดไม่ได้อีกตลอดกาล

GGameover303 committed:
CanSleep

- ตอนนี้ fade มี 2 ตัว
- fadingScreen ใช้แค่ SuperMarket
- FaderScreen จะใช้เป็นตัวหลักหลังจากนี้

GGameover303 committed:
ADD newVideoC3 And Target 64bitEverthing

beersanp commented:
um, all code is merged

มีปัญหานิดหน่อย
- เหมือนว่า ฉากแรกมัน fade ดำเองตอนรวมงาน (need recheck)
- ส่วนปุ่มที่แก้ไปใน #34 ทำงานได้ปกติ
- ละก็ ui มันไม่เลื่อนลงหลังจากกล่องข้อความหาย

รวมๆแล้วเหลือเอา code มาทำงานต่อกัน 
ถ้าพรุ่งนี้แก้ไม่ทัน น่าจะส่งให้ game test เล่นไม่ทันแหะ

beersanp commented:
อันนี้สงสัย https://github.com/khonsaiinc/aquamarine/blob/f552ac8600fe0e82544c9131f73af0a27b2c7b38/Assets/Script/DontMoveGlobal.cs

คือย้ายออกมาจาก dialoguemanager มาแล้วใช่มะ

beersanp committed:
fixed conflict

Co-authored-by: IFM <roon25552555@gmail.com>

update from https://github.com/khonsaiinc/aquamarine/pull/37 it's worked

issue
- blank dialogue, if script has choice selection

beersanp commented:
เสร็จละ ฝากเจ้าของมา recheck พรุ่งนี้ด้วย จะกด merge ให้

roon2555 committed:
UI in/out on outside scene

roon2555 reviewed (approved):
ไม่มีปัญหาอะไร
